RubberduckTests/Mocks/MockVbeBuilder.cs

```diff
@@ -16,10 +17,10 @@ namespace RubberduckTests.Mocks
     [SuppressMessage("Microsoft.Design", "CA1001")] //CA1001 is complaining about RubberduckTests.Mocks.Windows, which doesn't need to be disposed in this context.
     public class MockVbeBuilder
     {
-        public const string TestProjectName = "TestProject1";
-        public const string TestModuleName = "TestModule1";
+        public static string TestProjectName { set; get; } = "TestProject1";
+        public static string TestModuleName { set; get; } = "TestModule1";
```

Comment:
I think this is not a good idea because it potentially causes coupling between different tests. (The statics are shared between all tests.)

I think to achieve the goal of allowing to change the project name for VBEs constructed via static helper methods on this class it would be better to simply provide new overloads allowing to pass in this data.